### ISSUE TITLE
connectToStores decorator pattern failed to bind

### ISSUE THREAD

**Issue by jimzhan** [yahoo/fluxible#180]
Exactly same as the [docs](http://fluxible.io/api/addons/connectToStores.html)

Regular way works fine as expected but the decorator failed with:

```javascript
Uncaught TypeError: Cannot read property '__reactAutoBindMap' of undefined
```

**Comment by redonkulus** [comment]
@jimzhan we actually haven't released the new version of fluxible (0.5) which will include the decorator support. I will make a note of that in the docs.

**Comment by redonkulus** [comment]
This is now released in 0.5 and should not be an issues anymore.
